components/integrations/installed_outgoing_webhook.tsx

```diff
@@ -159,25 +199,17 @@ export default class InstalledOutgoingWebhook extends React.PureComponent<Props>
 
         let actions = null;
         if (this.props.canChange) {
-            actions = (
-                <div className='item-actions'>
+            actions = (this.state.hovering ?
```

Comment:
this seems like a pretty big re-render just for a mouse hover event. Perhaps you can use the same component and just update classes using `classNames`

Reply:
better use `:hover` selector and hide unwanted components